engine/runner/src/main/java/org/enso/runner/Main.java

```diff
@@ -1383,7 +1383,7 @@ private void launch(Options options, CommandLine line, Level logLevel, boolean l
                 return BoxedUnit.UNIT;
               });
         } catch (IOException ex) {
-          System.err.println(ex.getMessage());
+          ex.printStackTrace();
```

Comment:
`getMessage` is sometimes unhelpful for Java exceptions, e.g. `NoSuchFileException::getMessage` returns just the path - but no comment - so from `getMessage` you have no idea what happened, what kind of exception this was - you just see a file path and wonder where did this come from...

The stack trace should be more helpful I think.

Reply:
If so, the full stacktrace should be printed only when verbose mode is on. We don't want to show this to end users.

Reply:
What is the alternative? As mentioned, `getMessage` is utterly useless. I guess I could try `toString`.